src/main.jsx

```diff
@@ -1,10 +1,13 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App'
-import './index.css'
+import React from "react";
```

Comment:
Because you are using React version 18, you don't need to import React. So you can remove it.